Formula/coccinelle.rb

```diff
@@ -1,9 +1,10 @@
 class Coccinelle < Formula
   desc "Program matching and transformation engine for C code"
   homepage "http://coccinelle.lip6.fr/"
-  url "http://coccinelle.lip6.fr/distrib/coccinelle-1.0.6.tgz"
-  sha256 "8452ed265c209dae99cbb33b67bc7912e72f8bca1e24f33f1a88ba3d7985e909"
-  revision 1
+  url "https://github.com/coccinelle/coccinelle.git",
+      :tag      => "1.0.8",
+      :revision => "d678c34afc0cfb479ad34f2225c57b1b8d3ebeae"
+  head "https://github.com/coccinelle/coccinelle.git"
 
   bottle do
     rebuild 1
```

Comment:
can be removed

Reply:
Don't we usually not touch bottle blocks?

Reply:
The bottle block needs to stay, the `rebuild 1` can go. That's basically a `revision 1` when you don't want to update it for everyone.

Reply:
BrewTestBot seems to handle it: https://github.com/BrewTestBot/homebrew-core/commit/51f1bb96f059c5e865420c31f76ea24c493c6251

Reply:
Hmmm, TIL